src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
+        <b-container class="index-container">
```

Comment:
Can you show the nested structure of the HTML components in the alignments (at least 4 spaces)?
For example:

```
<b-container class="index-container">
    <b-row>
        <b-col>
            <h1>Mentorship...for you!</h1>
       </b-col>
    </b-row>     
</b-container>
```